data/pg/common-masters/uiHomePage.json

```diff
@@ -0,0 +1,134 @@
+                        "code": "CITIZEN_SERVICE_WS",
+                        "name": "Water & Sewerage",
+                        "label": "ACTION_TEST_WATER_AND_SEWERAGE",
+                        "enabled": true,
+                        "navigationUrl": "/digit-ui/citizen/ws-home"
+                    }
+                ]
+            },
+            "informationAndUpdatesCard":
+            {
+                "code": "HOME_CITIZEN_INFO_UPDATE_CARD",
+                "name": "Home Citizen Information and Updates card",
+                "enabled": true,
+                "headerLabel": "CS_COMMON_DASHBOARD_INFO_UPDATES",
+                "sideOption": {
+                    "name": "DASHBOARD_VIEW_ALL_LABEL",
+                    "enabled": true,
+                    "navigationUrl": ""
+                  },
+                "props": [
+                    {
+                        "code": "CITIZEN_MY_CITY",
+                        "name": "My City",
+                        "label": "CS_HEADER_MYCITY",
+                        "enabled": true,
+                        "navigationUrl": ""
+                    },
+                    {
+                        "code": "CITIZEN_EVENTS",
+                        "name": "Events",
+                        "label": "EVENTS_EVENTS_HEADER",
+                        "enabled": true,
+                        "navigationUrl": "/digit-ui/citizen/engagement/events"
+                    },
+                    {
+                        "code": "CITIZEN_DOCUMENTS",
+                        "name": "Documents",
+                        "label": "CS_COMMON_DOCUMENTS",
+                        "enabled": true,
+                        "navigationUrl": "/digit-ui/citizen/engagement/docs"
+                    },
+                    {
+                        "code": "CITIZEN_SURVEYS",
+                        "name": "Surveys",
+                        "label": "CS_COMMON_SURVEYS",
+                        "enabled": true,
+                        "navigationUrl": "/digit-ui/citizen/engagement/surveys/list"
+                    }
+                ]
+            },
+            "whatsNewSection":
+            {
+                "code": "WHATSNEW",
+                "name": "What's New",
+                "enabled": true,
+                "headerLabel": "DASHBOARD_WHATS_NEW_LABEL",
+                "sideOption": {
+                    "name": "DASHBOARD_VIEW_ALL_LABEL",
+                    "enabled": true,
+                    "navigationUrl": "/digit-ui/citizen/engagement/whats-new"
+                  }
+            },
+            "whatsAppBannerDesktop":
+            {
+                "code": "WHATSAPP_BANNER_DESKTOP",
+                "name": "WhatsApp Banner Desktop View",
+                "bannerUrl": "https://uat.digit.org/egov-uat-assets/app-banner-mobile.jpg",
+                "enabled": true,
+                "navigationUrl": "https://api.whatsapp.com/send?phone=918744060444&text=mSeva"
```

Comment:
Not sure whether the WhatsApp chatbot for UAT env uses same whatsapp number

Reply:
its the same phone number i did check in uat.yaml fle